webapp/sources/rudder/rudder-web/src/main/webapp/javascript/rudder/rudder.js

```diff
@@ -37,6 +37,22 @@
 
 /* Event handler function */
 
+var entityMap = {
+  '&': '&amp;',
+  '<': '&lt;',
+  '>': '&gt;',
+  '"': '&quot;',
+  "'": '&#39;',
+  '/': '&#x2F;',
+  '`': '&#x60;',
+  '=': '&#x3D;'
+};
+
+function escapeHTML (string) {
```

Comment:
could you also remove safeTagsReplace with that (cf https://github.com/Normation/rudder/pull/4390/files )